website/docs/components/pagination/partials/guidelines/guidelines.md

```diff
@@ -45,4 +45,10 @@ If your product uses a significantly higher or lower spacing scale, increase or
 
 ---
 
-![Spacing for pagination paired with not contained content](/assets/components/pagination/pagination-spacing-not-contained.png)
\ No newline at end of file
+![Spacing for pagination paired with not contained content](/assets/components/pagination/pagination-spacing-not-contained.png)
+
+### Pagination and filtering
+
+While pagination can be beneficial when dividing up a database into more managable chunks, relying _solely_ on pagination and sorting to find a specific record or set of records results in a poor user experience. This is esepcially true in cursor-based pagination where it may not be clear to the user where they are relatively within a data set.
```

Comment:
```suggestion
While pagination can be beneficial for dividing up and displaying a large dataset into more manageable chunks, relying _solely_ on pagination and sorting to find a specific record or set of records results in a poor user experience. This is especially true in cursor-based pagination, where it may not be clear to the user where their relative position is within the dataset.
```